config/config.go

```diff
@@ -83,10 +84,12 @@ type Config struct {
 
 	Routing RoutingC
 
-	EnableAutoRelay bool
-	Reachability    network.Reachability
-	AutoNATService  bool
-	StaticRelays    []peer.AddrInfo
+	EnableAutoRelay   bool
+	Reachability      network.Reachability
+	AutoNATService    bool
+	AutoNATThrottling bool
+	AutoNATThrottles  [2]int
```

Comment:
Let's use a struct instead of an array. Really, we should have one struct for all `AutoNATService` options as well.